gems/quilt_rails/README.md

```diff
@@ -400,30 +400,9 @@ By default, sewing-kit bundles in `@shopify/react-server-webpack-plugin` for `qu
       └─- server.{js|ts}x
 ```
 
-```tsx
-// app/ui/server.tsx
-import '@shopify/polyfills/fetch';
-import {createServer} from '@shopify/react-server';
-import {Context} from 'koa';
-import React from 'react';
-
-import App from './app';
-
-// The simplest way to build a custom server that will work with this library is to use the APIs provided by @shopify/react-server.
-// https://github.com/Shopify/quilt/blob/master/packages/react-server/README.md#L8
-const app = createServer({
-  port: process.env.PORT ? parseInt(process.env.PORT, 10) : 8081,
-  ip: process.env.IP,
-  assetPrefix: process.env.CDN_URL || 'localhost:8080/assets/webpack',
-  render: (ctx, {locale}) => {
-    const whatever = /* do something special with the koa context */;
-    // any special data we add to the incoming request in our rails controller we can access here to pass into our component
-    return <App server someCustomProp={whatever} location={ctx.request.url} locale={locale} />;
-  },
-});
+Use [`@shopify/react-server`](../../packages/react-server/README.md) to create a simple custom node server.
 
-export default app;
-```
+Alternatively, if no customization on the server is needed, [`@shopify/react-server-webpack-plugin`](../../packages/react-server-webpack-plugin/README.md) can be use to have a virtual server created for you.
```

Comment:
Move to linking the react-server doc. So the example it reference will always be up to date.

Reply:
Thanks for updating the docs on this so quickly! Two thoughts:
- I find the instructions here a bit confusing. Won't quilt_rails set up the node server for you? Therefor, if you don't need to customize it you won't need to do anything right?
- Secondly, I feel like a missing piece here is why you might want to customize it. In my case I'm trying to get the locale to pass it into the i18n manager which doesn't seem to be in there by default. I feel like for common use-cases such as that we should have some kind of docs that explain how to do that.

Reply:
* You are right. `quilt_rails` itself does not give you a node server, but by using `sewing-kit` it will automatically install `@shopify/react-server-webpack-plugin` and use it for you. So it does seem automatic from the outside, I just completely forgot about it 😬 

* I like the idea of doing common use-cases. Locale is not one thou as I think you can do it without. I will do some research but the only real need for customization I know of is web doing crazy auth things. But they dont use `quilt_rails`